### PR TITLE
[RTE][DRAFT] Fix CC for beta-release

### PR DESCRIPTION
**Pull request by palatter**
# What

# Why

# How Tested

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

File: packages/react-components/src/components/ChatMessage/MyMessageComponents/ChatMessageComponentAsRichTextEditBox.tsx [hunk 1]
```diff
@@ -283,7 +283,7 @@ export const ChatMessageComponentAsRichTextEditBox = (
         <RichTextSendBoxErrors
           textTooLongMessage={textTooLongMessage}
           systemMessage={message.failureReason}
-          /* @conditional-compile-remove(file-sharing-acs) */ attachmentUploadsPendingError={
+          /* @conditional-compile-remove(rich-text-editor-image-upload) */ attachmentUploadsPendingError={
```
Comment (vhuseinova-msft) [inline packages/react-components/src/components/ChatMessage/MyMessageComponents/ChatMessageComponentAsRichTextEditBox.tsx:286]: Should we have both?

File: packages/react-components/src/components/utils/SendBoxUtils.ts [hunk 2]
```diff
@@ -166,8 +167,8 @@ export const toAttachmentMetadata = (
  */
 export const insertAttachmentsAndImages = (
   content: string,
-  attachments: AttachmentMetadataInProgress[] | undefined,
-  imageUploadsInProgress: AttachmentMetadataInProgress[] | undefined
+  attachments?: AttachmentMetadataInProgress[] | undefined,
```
Comment (vhuseinova-msft) [inline packages/react-components/src/components/utils/SendBoxUtils.ts:170]: undefined is not needed when we use '?' after the parameter

File: packages/react-components/src/components/utils/SendBoxUtils.ts [hunk 3]
```diff
@@ -166,8 +167,8 @@ export const toAttachmentMetadata = (
...
+  imageUploadsInProgress?: AttachmentMetadataInProgress[] | undefined
 ): { content: string; attachmentArray: AttachmentMetadata[] | undefined } => {
```
Comment (vhuseinova-msft) [inline packages/react-components/src/components/utils/SendBoxUtils.ts:172]: Should we update `attachmentArray` here to have the same format?

File: packages/react-components/src/components/RichTextEditor/RichTextSendBox.tsx [hunk 4]
```diff
@@ -323,12 +324,14 @@ export const RichTextSendBox = (props: RichTextSendBoxProps): JSX.Element => {
     ) {
       /* @conditional-compile-remove(file-sharing-acs) */
       /* @conditional-compile-remove(rich-text-editor-image-upload) */
-      const { content, attachmentArray } = insertAttachmentsAndImages(
-        contentValue,
-        /* @conditional-compile-remove(file-sharing-acs) */ attachments,
-        /* @conditional-compile-remove(rich-text-editor-image-upload) */ imageUploadsInProgress
-      );
+      const { content, /* @conditional-compile-remove(file-sharing-acs) */ attachmentArray } =
```
Comment (Leah-Xia-Microsoft) [inline packages/react-components/src/components/RichTextEditor/RichTextSendBox.tsx:327]: I don't think we need this CC

**Pull request closed without merging**